hs/src/Reach/Backend/JS.hs

```diff
@@ -1022,6 +1022,7 @@ jsPIProg cr (PLProg _ _ dli dexports (EPPs {..}) (CPProg _ vi _ devts _)) = do
   partsp <- mapM (uncurry (jsPart dli)) $ M.toAscList epps_m
   cnpsp <- mapM (uncurry jsCnp) $ HM.toList cr
   let connMap = M.fromList [(name, "_" <> pretty name) | name <- HM.keys cr]
+  let ssmDoc = M.map (pretty . (\s -> "\"" ++ s ++ "\"") . show) ssm
```

Comment:
There's a function in this file for formatting "assertion info" that you should use